.circleci/config.yml

```diff
@@ -38,8 +38,6 @@ x-config:
       name: Embed environment variables into the app
       command: |
         touch .env.js
```

Comment:
Is anything inside this file anymore?

Reply:
No, but … something may come back in the future?

Reply:
Alright. I think one use case we'll have for it is when the map transitions to mapbox. So we can keep it around.